uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/api/addon/EventFilter.java

```diff
@@ -35,6 +35,7 @@ public final class EventFilter {
     private final Integer messageExpiryInterval;
     private final String responseTopic;
     private final byte[] correlationData;
+    private final Boolean isMessageFull;
```

Comment:
Changes in EventFilter is not required for that task.

Reply:
deleted in 29f79187174dd5710c02c749df6632e330bc6cf2